src/hex_api_key.erl

```diff
@@ -36,7 +36,7 @@
 %%           <<"user_agent">> => <<"hex_core/0.5.0 (httpc) (OTP/21) (erts/10.2)">>},
 %%     <<"name">> => <<"hex_core">>,
 %%     <<"permissions">> => [#{<<"domain">> => <<"api">>,<<"resource">> => <<"read">>}],
-%%     <<"revoked_at">> => nil,
+%%     <<"revoked_at">> => undefined,
```

Comment:
is this really `undefined`? I believe the backend returns `nil` and we pass that through?

Reply:
Actually yes it comes back as nil, and if there is no translation done in hex_core from nil to undefined, then this PR seems invalid. Not terribly sure it's a good idea putting said translation in 🤔